pkgs/applications/system/zxfer/default.nix

```
@@ -0,0 +1,51 @@
+{ lib
+, fetchFromGitHub
+, installShellFiles
+, makeWrapper
+, coreutils
+, gawk
+, gnugrep
+, gnused
+, rsync
+, stdenv
+, zfs
+}:
+
+stdenv.mkDerivation rec {
+  pname = "zxfer";
+  version = "1.1.7";
+
+  src = fetchFromGitHub {
+    owner = "allanjude";
+    repo = pname;
```

```suggestion
    repo = "zxfer";
```

It is not so much useful to parameterize the pname.

---

I actually find it to be very helpful at reducing typos

---

I actually find it to be very helpful at reducing typos

---

How many times this attribute will change? One or two in a decade, I suppose? This is too few to be afraid of typos.

But whatever.